types/prettier/index.d.ts

```diff
@@ -478,23 +478,25 @@ export namespace util {
     function addTrailingComment(node: any, commentNode: any): void;
 }
 
-// https://github.com/prettier/prettier/blob/master/src/doc/index.js
+// https://github.com/prettier/prettier/blob/main/src/document/index.js
 export namespace doc {
```

Comment:
@fisker Seems the `doc` folder was renamed :thinking:  
Do we need to rename the `namespace`?

Reply:
No, checked it, it's still `prettier/doc` :+1: